Libplanet.Tests/Blocks/BlockTest.cs

```diff
@@ -211,10 +214,13 @@ public void DetectInvalidProtocolVersion()
                 new Transaction<DumbAction>[] { },
                 protocolVersion: -1
             );
-            Assert.Throws<InvalidBlockProtocolVersionException>(() => block.Validate(now));
+            Assert.Throws<InvalidBlockProtocolVersionException>(
+                () => block.Validate(_fx.GetHashAlgorithm(_fx.Next.Index), now)
```

Comment:
We may need `ValidationContext` to consolidate the hash algorithm and DateTime... 🤔

Reply:
As `HashAlgorithmGetter` is usually from `IBlockPolicy.GetHashAlgorithm`, even if we make a new data type its construction would be unnatural IMHO.